security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -18,6 +18,18 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
 For more information on compliance support for all Red{nbsp}Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
 
 
+[id="compliance-operator-release-notes-1-7-1_{context}"]
+== OpenShift Compliance Operator 1.7.1
+
+The following advisory is available for the OpenShift Compliance Operator 1.7.1:
+
+* link:https://access.redhat.com/errata/RHBA-YYYY:NNNN[RHBA-YYYY:NNNN - OpenShift Compliance Operator 1.7.1 bug fix update]
+
+[id="compliance-operator-1-7-1-bug-fixes_{context}"]
+=== Bug fixes
+
+* Previously, the Compliance Operator's `pauser` container could be terminated due to running out of memory (OOMKilled). This behavior particularly affects {product-title} 4.16.z clusters. With this update, the memory limit for the `pauser` container is increased to prevent the error and improve overall stability. (link:https://issues.redhat.com/browse/OCPBUGS-50924[*OCPBUGS-50924*])
```

Comment:
Since we have the same code for all releases, I think you can remove `This behavior particularly affects {product-title} 4.16.z clusters.` All others lgtm